flaml/automl.py

```diff
@@ -2315,7 +2315,7 @@ def _search_parallel(self):
             ),
             key=lambda x: x.last_result["wall_clock_time"],
         )
-        for _track_iter, trial in enumerate(trials):
+        for self._track_iter, trial in enumerate(trials):
```

Comment:
This makes line 571 redundant?

Reply:
line 571 is needed for search_sequential()